js/plugins/omniture.js

```diff
@@ -162,6 +162,7 @@ var OmnitureAnalyticsPlugin = function (framework)
       //TODO: Get Player version
       ahPluginConfig.sdk = OOYALA_PLAYER_VERSION;
       ahPluginConfig.debugLogging = metadata.debug; // set this to false for production apps.
+      if (metadata.heartbeatSSL) ahPluginConfig.ssl = metadata.heartbeatSSL; // set this to true to enable Heartbeat calls through HTTPS
```

Comment:
Can we have these on separate lines? This will make it consistent with our style guidelines and will make it easier to debug.